### PR TITLE
Update with code generation

### PR DESCRIPTION
**Pull request by raddevon**
Companion PR to edgedb/edgedb#4709

### CODE REVIEW

**Comment by raddevon** [comment]
Want to get eyes on this to make sure my Python code is sound. If I need to make any changes here, those will likely cascade over to the guide update as well.

File: fastapi-crud/app/events.py [hunk 1]
```diff
@@ -25,8 +27,8 @@ class Host(BaseModel):
 
 class ResponseData(BaseModel):
     name: str
-    address: str
-    schedule: datetime.datetime
+    address: Optional[str]
+    schedule: Optional[datetime.datetime]
     host: Optional[Host]
```
Comment (fantix) [inline fastapi-crud/app/events.py:32]: I think the idea here is to use e.g. `dbqueries.CreateEventResult` directly without having to wrap the result again.

**Review by fantix — CHANGES_REQUESTED** [review]
Other than the response models, this looks good to me!

File: fastapi-crud/app/events.py (hunk 1 above)
Comment (raddevon) [reply fastapi-crud/app/events.py:32]: Thanks for the note on this. I will circle back and use the generated model.

Comment (fantix) [reply fastapi-crud/app/events.py:32]: Thanks and sorry for the late notice! As you're using single-file mode, I think your generated model names are sufferring from [this issue](https://github.com/edgedb/edgedb-python/issues/397) too, it's probably better to use something like `import ... CreateEventResult as ...EventResult` (or `EventResult = db_queries.CreateEventResult`) to rename the models for now.

Comment (raddevon) [reply fastapi-crud/app/events.py]: Just pushed some changes addressing this, @fantix. Check it out when you have a chance and let me know if I need to revisit.

**Review by fantix — APPROVED** [review]

**Pull request merged**